cmd/homi/setup/flags.go

```diff
@@ -290,6 +295,12 @@ var (
 		Value: params.DefaultRatio,
 	}
 
+	rewardKip82RatioFlag = cli.StringFlag{
+		Name:  "reward-kip82ratio",
```

Comment:
When a variable is mapped to the Name, 
if the pattern ( capital letters are changed to - ) is applied here as well, 
it would be nice to be consistent. 

Name: "reward-kip82-ratio",

Reply:
fixed in a92cea52